docs/myriad-configuration.md

```diff
@@ -63,10 +68,14 @@ executor:
   jvmMaxMemoryMB: 256   # Xmx for myriad's executor that launches Node Manager.
   path: file://localhost/usr/local/libexec/mesos/myriad-executor-0.0.1.jar  # Path for the myriad's executor binary.
                                                                             # Also supports, hdfs:// notation.
+  #These are for remote distribution
```

Comment:
Since this is the configuration doc, could you please elaborate about how these parameters are used to configure remote distribution?
Nit: Can you put a space after the '#', please?